airbyte-server/build.gradle

```diff
@@ -3,6 +3,14 @@ plugins {
 }
 
 dependencies {
+    constraints {
+        implementation 'com.github.jnr:jnr-posix:3.1.8'
```

Comment:
Some of these are duplicated on multiple subprojects. Can these constraints be applied globally?

Reply:
Not sure. We'll merge all the approved PRs to rescan and get a feel for what's remaining. I'll try to address this once we've fixed all the vulnerabilities.